client/question/domain/SpeechBalloonObjectFactory.js

```diff
@@ -67,6 +67,15 @@ tie.factory('SpeechBalloonObjectFactory', [
       return this._type === SPEECH_BALLOON_TYPE_FEEDBACK;
     };
 
+    /**
+     * Returns whether the code balloon is displayed on the right.
```

Comment:
Update comment to reflect name change (see below)

Reply:
Fixed

Reply:
Nit: Returns whether or not the current speech balloon is a code submission balloon.

Not gonna block submit, though, so change that and you're good to go.

Reply:
This does not appear to have been changed (see @eyurko's last comment).

Reply:
Hm, seems that the latest commit did not get merged in. I can make a new pull request for this.